shell/renderer/api/electron_api_context_bridge.cc

```diff
@@ -417,6 +417,50 @@ v8::MaybeLocal<v8::Value> PassValueToOtherContextInner(
       return v8::MaybeLocal<v8::Value>(
           blob.ToV8Value(destination_context->GetIsolate()));
     }
+
+    // Custom logic to "clone" FileList references
+    auto object_value = value.As<v8::Object>();
+    v8::Maybe<bool> has_length = object_value->Has(
+        source_context,
+        gin::StringToV8(source_context->GetIsolate(), "length"));
+    v8::Maybe<bool> has_item = object_value->Has(
+        source_context, gin::StringToV8(source_context->GetIsolate(), "item"));
```

Comment:
- When you've got a string literal, `v8::String::NewFromUtf8Literal()` is fastest
- We v8ize these two keys twice, but only need to do it once.

```suggestion

    const v8::Local<v8::String> length_key =
        v8::String::NewFromUtf8Literal(source_context->GetIsolate(), "length");
    const v8::Local<v8::String> item_key =
        v8::String::NewFromUtf8Literal(source_context->GetIsolate(), "item");

    v8::Maybe<bool> has_length = object_value->Has(source_context, length_key);
    v8::Maybe<bool> has_item = object_value->Has(source_context, item_key);
```